react-hyf/client/src/App.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import './App.css';
+
+// COMPONENTS
+import NavBar from './components/navBar/NavBar';
+import Home from './components/home/Home';
+import StaticList from './components/week1/StaticList';
+import DynamicTodoApp from './components/old_homeworks/DynamicTodoApp';
+import HomeworkWeek3 from './components/week3_homework/HomeworkWeek3';
+import WrongUrl from './components/wrongUrl/WrongUrl';
+
+class App extends Component {
+  render() {
+    return (
+      <div>
+        <Router>
+          <NavBar />
```

Comment:
Put `Router` in the very top of your application which is `index.js` otherwise you will not be able to use `withRouter` in `App.js`

Reply:
done..